crowdsec-docs/docs/user_guides/alert_context.md

```diff
@@ -4,34 +4,46 @@ title: Contextualize Alerts
 sidebar_position: 10
 ---
 
+# Alert Contextualization
 
+While CrowdSec doesn't store any logs after processing it, it can be useful to get some context about why an alert has been triggered (source machine, targeted FQDN ...).
 
-# Alert Contextualization
+The context configuration is part of the CrowdSec Hub and each collection should come with a context configuration file.
 
+The only thing to do if you want to see the context in the console is to enable the `context` option in your CrowdSec Local API (see below).
 
-While CrowdSec doesn't store any logs after processing it, it can be useful to get some context about why an alert has been triggered (source machine, targeted FQDN ...).
+It is also possible to choose which fields from a parsed log you want to be added in the context of the alert directly in your Security Engine.
```

Comment:
Meta can also be part of the context (not all metas are coming the log line itself).
And the value can also be a static string in the context configuration.